README.md

```diff
@@ -1193,6 +1193,27 @@ There would also be logging to the standard out stream as well.
 
 The `cromwell.<date>.log` file contains an aggregate of every log message, while the `workflow.<uuid>.log` files contain only log messages that pertain to that particular workflow.
 
+# Call Caching
+
+Call Caching allows Cromwell to re-use the outputs of a call if the exact same call is about to be run again. 
+**Call Caching is disabled by default.**
+
+To enable Call Caching, add the following to your Cromwell configuration:
+
+```
+call-caching {
+  enabled = true
```

Comment:
I'm not sure why you would want to disable "add-to-cache", I thought that this would be happening anyway and that the configuration would let you choose weather or not you "read-from-cache".
Agreed that the terminology should be more explicit though.